plugin/PDFView.qml

```diff
@@ -218,6 +228,18 @@ SilicaFlickable {
             }
         }
 
+        Rectangle {
+            x: linkArea.clickedBox.x
+            y: linkArea.clickedBox.y
+            width: linkArea.clickedBox.width
+            height: linkArea.clickedBox.height
+            radius: Theme.paddingSmall
+            color: Theme.highlightColor
+            opacity: 0.75
+            visible: linkArea.pressed
+            Behavior on opacity { FadeAnimation { duration: 100 } }
```

Comment:
Oups, that's right. I've slightly changed this Rectangle showing bindings during previous iterations but didn't notice this inconsistency… Corrected now.